net/ooniprobe/Makefile

```diff
@@ -0,0 +1,56 @@
+#
+# Copyright (C) 2020 CZ.NIC, z. s. p. o. (https://www.nic.cz/)
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=ooniprobe
+PKG_VERSION:=3.0.0-rc.14
+PKG_RELEASE:=1
+
+PKG_SOURCE:=probe-cli-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://codeload.github.com/ooni/probe-cli/tar.gz/v$(PKG_VERSION)?
+PKG_HASH:=b084e393bc9c869511657d4cedaacceb50fdd25cd324cf9b6bb47001cf622016
+
+PKG_MAINTAINER:=Jan Pavlinec <jan.pavlinec@nic.cz>
+PKG_LICENSE:=BSD-3-Clause
+PKG_LICENSE_FILES:=LICENSE.md
+
+PKG_BUILD_DIR:=$(BUILD_DIR)/probe-cli-$(PKG_VERSION)
+PKG_BUILD_DEPENDS:=golang/host measurement-kit libcurl
```

Comment:
Since libcurl is in `DEPENDS` below, is libcurl here necessary?

Reply:
In the end, libcurl was not necessary. When I tried that on new SDK compilation passed without it so there was some error in my environment.

Reply:
Actually, curl does need to be compiled before measurement-kit; I've opened #12217 to address this.